adapter_pipelines/submit.wdl

```diff
@@ -62,6 +62,7 @@ task create_submission {
   Int? individual_request_timeout
   Boolean record_http
   String pipeline_tools_version
+  Boolean add_md5s
```

Comment:
Should we also add this parameter to the stub submit wdl? Otherwise it may fail PRs in skylab i guess